common/src/main/java/com/wynntils/services/itemfilter/statproviders/territory/TerritoryNameStatProvider.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.services.itemfilter.statproviders.territory;
+
+import com.wynntils.models.items.items.gui.TerritoryItem;
+import java.util.List;
+
+public class TerritoryNameStatProvider extends TerritoryStatProvider<String> {
```

Comment:
Is this necessary for the ui? If not, does the plain query not handle the name  in some cases?

Reply:
The real search would also search for the (HQ)/[!] parts. Your question is good though, Ill think about it until tomorrow.

Reply:
Ah I see, are you wanting to have the names in the valid inputs list the filter ui uses? I see no reason to remove it if that's the case

Reply:
> Ah I see, are you wanting to have the names in the valid inputs list the filter ui uses? I see no reason to remove it if that's the case

Perhaps we could use it like that, yes.